control_panel_api/aws.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+import boto3
+
+
+def create_bucket(name, region, **kwargs):
```

Comment:
We may well (in this case with 3 args) to also add `acl='private'` to the list of arguments.